.github/workflows/icu_ci.yml

```diff
@@ -32,6 +32,31 @@ jobs:
           ant init;
           ant check;
 
+  # gcc debug build.
+  # Includes dependency checker.
+  # Note - the dependency checker needs to be run on both a debug and an optimized build.
+  # This one (gcc) for debug, and linux clang (see job below) for optimized.
+  #
+  # Test both out-of-source and in-source builds. This one (gcc) for out-of-source,
+  # and linux clang (below) for in-source.
+  #
+  # Invokes test/hdrtst to check public headers compliance.
+  gcc-debug-build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: ICU4C with gcc
+        run: |
+          mkdir build;
+          cd build;
+          ../icu4c/source/runConfigureICU --enable-debug --disable-release Linux/gcc --prefix=/tmp/icu-prefix --enable-tracing;
```

Comment:
optional: In the .travis.yml file, and in the job that you added right below this, you made `PREFIX` an env variable and reference it using `${PREFIX}`. You could do that here to for consistency's sake.

Reply:
Done.